### PR TITLE
added image similarity slider

### PR DESCRIPTION
**Pull request by swaaaaaathii**

### CODE REVIEW

File: src/ReverseImageSearchForm.js [hunk 1]
```diff
@@ -1,18 +1,67 @@
...
+	    	<Form.Label>Minimum image similarity level 
+	    		<Tooltip title="Image similarity lets you control how narrowly or broadly to match ad creatives images. Very High will match ad creative images nearly identical to the uploaded image. Medium will match ad creative images somewhat similar, very similar, and nearly identical to the uploaded image. Very Low will match ad creatives images anywhere between loosely similar and nearly identical to the uploaded image." placement="right">
+					<IconButton aria-label="info">
+						<MdInfo />
+					</IconButton>
+				</Tooltip>
+	    	</Form.Label>   	  
+		    <Slider
+				defaultValue={50}
+				getAriaValueText={valuetext}
```
Comment (macpd) [inline src/ReverseImageSearchForm.js]: it's good to provide value text for accessibility, but I think the setState call should be done through a callback passed to `onChange`

File: src/ReverseImageSearchForm.js [hunk 2]
```diff
@@ -1,18 +1,67 @@
...
+			value: 50,
+			label: 'Medium',
+		},
+		{
+			value: 75,
+			label: 'High',
+		},
+		{
+			value: 100,
+			label: 'Very High',
```
Comment (macpd) [inline src/ReverseImageSearchForm.js]: Please reverse the order of the labels. "very high" will return less results than "very low" so I'm hoping that having "very high" on the left and "very low" on right will indicate an increasingly larger set of results from left to right (for left to right readers). Does that make sense?

**Review by macpd — COMMENTED** [review]
This looks and works great. just a few requests

File: src/ReverseImageSearchForm.js (hunk 2 above)
Comment (swaaaaaathii) [reply src/ReverseImageSearchForm.js]: Yes it does. I'll make the change.

File: src/ReverseImageSearchForm.js (hunk 1 above)
Comment (swaaaaaathii) [reply src/ReverseImageSearchForm.js]: Okay, I'll work on this

**Review by macpd — APPROVED** [review]

**Pull request merged**